### PR TITLE
July UI, add support for a second speed bump, new annotation to avoid running some tests on daily

### PR DESCRIPTION
**Pull request by fadidurah**
July Ui maintentance, some cross cloud tests are experiencing a second speed bump page, also created an annotation to signify cases that would fail if ran with daily versions (due to the versions starting with 0.0).

### CODE REVIEW

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/AadLoginComponentHandler.java [hunk 1]
```diff
@@ -212,6 +212,9 @@ public void handleVerifyYourIdentity() {
     public void handleHowWouldYouLikeToSignIn() {
         // Looks like we sometimes see this UI prompt asking "How would like to sign in?"
         // We press button1, which is "Ok" to confirm the default selection of using device certificate.
-        UiAutomatorUtils.handleButtonClick("android:id/button1", mFindLoginUiElementTimeout);
+        UiAutomatorUtils.handleButtonClickSafely("android:id/button1", mFindLoginUiElementTimeout);
+
+        // Sometimes need to also confirm the selection
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/AadLoginComponentHandler.java]: Is this going to attempt two times in every test case? So if there is not a need to confirm then code will still try to click this button?

Comment (fadidurah) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/AadLoginComponentHandler.java]: Yes, this probably should be another parameter. I'll add that.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/annotations/FailsWithDailyVersions.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import java.lang.annotation.Target;
+
+/**
+ * An annotation indicating that an automated End-to-End test case is failing when used with Daily Pipeline versions,
+ * but should still be attempted on Monthly Release Pipeline for validation.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+public @interface FailsWithDailyVersions {
+    String value() default "";
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/annotations/FailsWithDailyVersions.java:37]: Why is a value needed?

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/PromptHandlerParameters.java [hunk 3]
```diff
@@ -133,6 +139,11 @@ public class PromptHandlerParameters {
      */
     private final boolean howWouldYouLikeToSignInExpected;
 
+    /**
+     * Denotes whether or not to except "Choose certificate" Prompt
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/PromptHandlerParameters.java]:
```suggestion
     * Denotes whether or not to expect "Choose certificate" Prompt.
```

**Review by shahzaibj — APPROVED** [review]
Left some comments but otherwise lgtm

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/annotations/FailsWithDailyVersions.java (hunk 2 above)
Comment (fadidurah) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/annotations/FailsWithDailyVersions.java:37]: Just to give room to provide reasoning for adding the annotation

**Review by rpdome — APPROVED** [review]

**Pull request merged**